config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/dishes/:id", to: "dishes#show"
+  patch "/dishes/:id", to: "dishes#update"
```

Comment:
As mentioned above, a more RESTful option for these operations would be:
`post "/dishes/:dish_id/ingredients", to: "dish_ingredients#create"`
`delete "/dishes/:dish_id/ingredients/:id", to "dish_ingredients#destroy"`
Note how passing these IDs in the path will prevent the need to pass them as parameters with buttons/forms which is helpful!